abstracts/abstract_en.tex

```diff
@@ -9,14 +9,17 @@
     On the other hand, a recent game \textit{Hyperbolica} popularized a novel idea of making the virtual worlds even more interesting by setting them in non-Euclidean spaces.
     This work describes the implementation of a video game that incorporates both of the aforementioned concepts.
 
-    In the game a technique of transforming Euclidean space into a hyperbolic or spherical space was used, which allows for the use of established algorithms.
-    This approach simplifies the implementation of the game logic in different geometries as it allows sharing large parts of the implementation across the three geometries.
-    It also allows for changing the degree to which the scene is visually curved in hyperbolic geometry at runtime, which is the best way to showcase the differences between the geometries.
+    In the game, we use and expand on the technique of transforming Euclidean space into hyperbolic or spherical space introduced by László Szirmay-Kalos and Milán Magdics in \cite{Szirmay-Kalos2022}.
+    The approach allows for the use of established algorithms, developed with Euclidean geometry in mind, in other geometries.
+    %This approach also simplifies the implementation of the game logic in different geometries as large parts of the implementation can be shared across the three geometries.
```

Comment:
I think this can be removed

Reply:
Do you mean the commented-out part only?

Reply:
yes